app/portainer/components/form-components/file-upload-form/file-upload-form.html

```diff
@@ -1,19 +1,11 @@
 <ng-form class="file-upload-form" name="$ctrl.fileUploadForm">
```

Comment:
are we not replacing file-upload-form with FileUploadField?

Reply:
that was my initial idea but I wasn't able to easily replace the `ng-transclude=description` part of the angular component, and this wasn't a blocker for intel so I split into two separate tickets

Reply:
ng-transclude is not support by react2angular. the matching feature in react would be a render prop `description` that receives a `ReactNode`.

please implement this so we won't miss it.

in cases where we have transcule we are forced to have a react component and an angular one.